src/main.rs

```diff
@@ -660,12 +662,17 @@ async fn filter_message_edit(update: &MessageUpdate, state: &State) -> Result<()
                 }
             };
 
+            let clean_message_content = crate::message::clean_mentions(
+                content,
+                update.mentions.as_ref().unwrap_or(&vec![]),
```

Comment:
this avoids creating a temporary `Vec` (I think it gets optimized away regardless but still)
```suggestion
                update.mentions.as_deref().unwrap_or(&[]),
```